### PR TITLE
Add child issues to Jira extension

### PR DESCRIPTION
**Pull request by horumyy**
## Description

Implemented the functionality to display child issues in the `IssueDetail.tsx` component. Additionally, a new option named “Open Child Issues” has been added to the action panel, enabling users to navigate through the child issues.

## Screencast

Screenshot 2024-11-03 at 2 14 56 p m
Screenshot 2024-11-03 at 2 15 13 p m

https://github.com/user-attachments/assets/118d10aa-4c6e-46f4-9f52-32c51d172c4e

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by horumyy** [comment]
Right now I'm getting an error after opening a IssueDetails from a IssueChildIssues, however the UI and data seems to be correct, in struggling with a jql error, if anyone have any recommendations i'm open to implement them

**Comment by horumyy** [comment]
Is there anything else that needs to be checked in this pr? or anything that needs to be corrected for merge? @thomaslombart

**Comment by horumyy** [comment]
> Yes, I left a comment just above.

are you sure? i dont get to see it

**Comment by horumyy** [comment]
> Yes, I left a comment just above.

sure? i dont see it

**Comment by thomaslombart** [comment]
Oh indeed, it was a pending comment, sorry! 😅

File: extensions/jira/src/components/IssueChildIssues.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  const {
+    issues: subtasks,
+    isLoading: isLoadingSubtasks,
+    mutate: mutateSubtasks,
+  } = useIssues(
+    subtaskJql || "issue = null", // Provide valid JQL even when no subtasks
+  );
+
+  const isEpic = issue.fields.issuetype?.name === "Epic";
+  const { issues: epicIssues, isLoading: isLoadingEpicIssues } = useEpicIssues(
```
[the remark this answers is not in the document]
Comment (horumyy) [reply extensions/jira/src/components/IssueChildIssues.tsx]: good catch, fixed on latest commit! :)

**Comment by horumyy** [comment]
Thank you for your comment c:, fixed!

**Comment by thomaslombart** [comment]
All good, thanks for the PR! 🙌

**Comment by horumyy** [comment]
> 🎉 🎉 🎉
> 
> We've rewarded your Raycast account with some credits. You will soon be able to exchange them for some swag.

:O this is so cool!, how do I redeem the credits?

**Pull request merged**